### PR TITLE
refactor: move prefixCls to wrapper

### PR DESCRIPTION
**Pull request by LongHaoo**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Prepare for CssInJs and place preClsName generation in the outer layer       |
| 🇨🇳 Chinese |     为CssInJs做准备，将preClsName生成放到外层      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
是不是放v5啊 next分支？

File: components/affix/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ function getDefaultTarget() {
 }
 
 // Affix
-export interface AffixProps {
+export interface AffixWrapperProps {
```
Comment (MadCcc) [inline components/affix/index.tsx:21]: 命名反了吧，有 `affixPrefixCls` 的不应该暴露给外部

Comment (LongHaoo) [reply components/affix/index.tsx:21]: 没反

Comment (LongHaoo) [reply components/affix/index.tsx:21]: @yoyo837 要做改造要在master

**Review by MadCcc — APPROVED** [review]

**Comment by zombieJ** [comment]
LGTM

File: components/affix/index.tsx [hunk 2]
```diff
@@ -286,7 +296,20 @@ class Affix extends React.Component<AffixProps, AffixState> {
...
 }
 
-const AffixFC = React.forwardRef<Affix, AffixProps>((props, ref) => <Affix {...props} ref={ref} />);
+const AffixFC = React.forwardRef<Affix, AffixProps>((props, ref) => {
+  const { prefixCls: customizePrefixCls } = props;
+  const { getPrefixCls } = React.useContext(ConfigContext);
+
+  const affixPrefixCls = getPrefixCls('affix', customizePrefixCls);
+
+  const AffixProps: InternalAffixProps = {
```
Comment (afc163) [inline components/affix/index.tsx:305]: AffixProps 应该是小写，类名和 interface 用大写。

File: components/affix/index.tsx [hunk 3]
```diff
@@ -33,6 +33,10 @@ export interface AffixProps {
   children: React.ReactNode;
 }
 
+export interface InternalAffixProps extends AffixProps {
```
Comment (afc163) [inline components/affix/index.tsx:36]: InternalAffixProps 的不应该 export 出去。

**Pull request merged**